packages/graph/hash_graph/lib/graph/src/store/mod.rs

```diff
@@ -327,11 +326,12 @@ pub trait EntityStore: crud::Read<Entity> {
     /// - if an [`EntityUuid`] was supplied and already exists in the store
     async fn create_entity(
         &mut self,
-        properties: EntityProperties,
+        entity_id: EntityId,
+        decision_time: Option<Timestamp>,
```

Comment:
Do we want to _consider_ making new types for DecisionTime and TransactionTime?

Reply:
I was thinking we _might_ also want a `DecisionTimeSpan` 😅

Reply:
Hmm, at least when exposing these this might be useful. I will consider that, thanks!

Reply:
I would prefer them being newtypes if it isn't problematic with these SQL types, but not a strong preference. Would suck to pass the wrong timespan because of the type